optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/ConferenceScheduling/ConferenceScheduling-section.adoc

```diff
@@ -17,16 +17,21 @@ Hard constraints:
 * Speaker unavailable timeslots: Every talk's speaker must be available during the talk's timeslot.
 * Speaker conflict: Two talks can't share a speaker during overlapping timeslots.
 * Generic purpose timeslot and room tags
-** Speaker required timeslot tag: If a speaker has a required timeslot tag, then all his/her talks must be assigned to a timeslot with that tag.
-** Speaker prohibited timeslot tag: If a speaker has a prohibited timeslot tag, then all his/her talks cannot be assigned to a timeslot with that tag.
-** Talk required timeslot tag: If a talk has a required timeslot tag, then it must be assigned to a timeslot with that tag.
-** Talk prohibited timeslot tag: If a talk has a prohibited timeslot tag, then it cannot be assigned to a timeslot with that tag.
-** Speaker required room tag: If a speaker has a required room tag, then all his/her talks must be assigned to a room with that tag.
-** Speaker prohibited room tag: If a speaker has a prohibited room tag, then all his/her talks cannot be assigned to a room with that tag.
-** Talk required room tag: If a talk has a required room tag, then it must be assigned to a room with that tag.
-** Talk prohibited room tag: If a talk has a prohibited room tag, then it cannot be assigned to a room with that tag.
-* Talk mutually-exclusive-talks tag: Talks that share such a tag must not be scheduled in overlapping timeslots.
+** Speaker required timeslot tags: If a speaker has a required timeslot tag, then all his/her talks must be assigned to a timeslot with that tag.
+** Speaker prohibited timeslot tags: If a speaker has a prohibited timeslot tag, then all his/her talks cannot be assigned to a timeslot with that tag.
+** Talk required timeslot tags: If a talk has a required timeslot tag, then it must be assigned to a timeslot with that tag.
+** Talk prohibited timeslot tags: If a talk has a prohibited timeslot tag, then it cannot be assigned to a timeslot with that tag.
+** Speaker required room tags: If a speaker has a required room tag, then all his/her talks must be assigned to a room with that tag.
+** Speaker prohibited room tags: If a speaker has a prohibited room tag, then all his/her talks cannot be assigned to a room with that tag.
+** Talk required room tags: If a talk has a required room tag, then it must be assigned to a room with that tag.
+** Talk prohibited room tags: If a talk has a prohibited room tag, then it cannot be assigned to a room with that tag.
 * Talk prerequisite talks: A talk must be scheduled after all its prerequisite talks.
+* Consecutive talks pause: A speaker who has more than one talk must have a break between them.
+
+Medium constraints:
+
+* Talk mutually-exclusive-talks tags: Talks that share such tags must not be scheduled in overlapping timeslots.
+* Published timeslot: A talk must not be scheduled at a different timeslot unless a hard constraint is broken.
```

Comment:
For clarity, let's change the description from "A talk ... a different timeslot ..." to "A publish talk ... a different timeslot than currently published ..."

Reply:
Let's change the "unless a hard constraint is broken" to a new sentence: If the hard constraint's input data changes after publishing (such as speaker availability), then this medium constraint will be minimally broken to attain a new feasible schedule."